README.md

```diff
@@ -39,6 +39,14 @@ You can use Budget by hosting it yourself, or using [the instance hosted by us](
 
 *Note that in order for certain features to work properly, the jobs queue needs to be watched. This can be done by either running `php artisan queue:work` or using [Supervisor](https://laravel.com/docs/7.x/queues#supervisor-configuration).*
 
+## Docker
+
+You can run Budget using Docker. Spinning it up (`docker-compose up`) will create 3 containers–for MySQL, PHP and NGINX.
+
+To run the migrations on the database, you can run `docker-compose exec php php artisan migrate`. Or, if you haven't installed the required dependencies, you can run `docker-compose exec php php artisan budget:install`.
```

Comment:
I would put the install command inside the Dockerfile.

Reply:
```diff
FROM php:7.2-fpm

RUN docker-php-ext-install pdo_mysql
+ 
+ CMD php artisan budget:install
```

Like this?

Reply:
I don't think it is necessary to install the dependencies at runtime. ``RUN`` would be better cause it caches the dependencies in the Docker Image.

Reply:
Fair enough, will update this.